enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/openshift/RouteEnricher.java

```diff
@@ -105,6 +100,18 @@ private RoutePort createRoutePort(ServiceBuilder serviceBuilder) {
         return routePort;
     }
 
+    private boolean hasWebPort(List<ServicePort> ports) {
```

Comment:
Why are we checking only a specific set of ports of web ports? What if user has supplied some other port rather than these ports? if he has, can we add that check here as well along with standard ports?

Reply:
I have followed what the expose enricher used to do.
Also, the purpose of a Route is basically to service incoming traffic..which comes through http and https only. So this set of ports looks legitimate to me.

 @lordofthejars thoughts ?

Reply:
I'm okay with these ports, but I'm just concerned about the case when the port is not among one of these ports

Reply:
Then that simply violates http and https and it should not be accepted, imo